app/components/elements/forms/field_component.rb

```diff
@@ -28,7 +28,7 @@ def help_text_id
       end
 
       def field_aria
-        return if @help_text.blank?
+        return if help_text.blank?
```

Comment:
Already have an `attr_reader` for this property, so use it instead of relying on the instance variable.